### PR TITLE
displaying user in lexicographic order

### LINKED ISSUE #4296: Change sort to lexicographic order on first name in Manage Users
### Is your proposal related to a problem?

Currently, on the manage users page, the users are listed according to the way they were added/invited.

### Describe the solution you'd like

Sort the first name according to lexicographic order

Screenshot 2022-10-04 at 4 17 57 PM

### PR DESCRIPTION
**Pull request by amit-ksh**
Fixes #4296

### CODE REVIEW

File: frontend/ee/components/UsersPage/UsersTable.jsx [hunk 1]
```diff
@@ -67,7 +78,7 @@ const UsersTable = ({
               <tbody>
                 {Array.isArray(users) &&
                   users.length > 0 &&
-                  users.map((user) => (
+                  sortUsersByFirstName(users).map((user) => (
```
Comment (shubh22) [inline frontend/ee/components/UsersPage/UsersTable.jsx:81]: Changes should be applied to the backend and not, the front end.
API call should return results in an alphabetically sorted format.

**Review by shubh22 — DISMISSED** [review]

File: frontend/ee/components/UsersPage/UsersTable.jsx (hunk 1 above)
Comment (amit-ksh) [reply frontend/ee/components/UsersPage/UsersTable.jsx:81]: Ok I'll do that give me some time

Comment (amit-ksh) [reply frontend/ee/components/UsersPage/UsersTable.jsx]: Made all the necessary changes! 😊

**Comment by amit-ksh** [comment]
I made all the suggested changes please review it @withshubh

**Review by amit-ksh — COMMENTED** [review]
Made all the requested changes

**Comment by gsmithun4** [comment]
deployed https://tj-4394.herokuapp.com/

**Comment by arpitnath** [comment]
@amit-ksh 
The UI seems to be broken.
Screenshot 2022-10-26 at 2 14 21 AM

**Review by arpitnath — CHANGES_REQUESTED** [review]
@amit-ksh 
The UI seems to be broken.
Screenshot 2022-10-26 at 2 14 21 AM

**Comment by amit-ksh** [comment]
Fixed the UI.

tooljet dashboard

**Review by arpitnath — APPROVED** [review]

**Comment by withshubh** [comment]
Hey @amit-ksh  
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**